code/modules/events/grid_check.dm

```diff
@@ -1,16 +1,18 @@
 /datum/event/grid_check	//NOTE: Times are measured in master controller ticks!
 	announceWhen		= 5
-
+	var/power_loss_sound = sound('sound/effects/powerloss.ogg')
```

Comment:
this should be `var/sound/s = sound('sound/effects/powerloss.ogg')`

and the whole thing should be done when `start` is called. We don't want to needlessly create sounds when we don't need them.